src/PlusImageProcessing/Testing/vtkPlusTransverseProcessEnhancerTest.cxx

```diff
@@ -21,43 +21,80 @@ This is a program meant to test vtkPlusTransverseProcessEnhancer.cxx from the co
 #include "vtkPlusTrackedFrameList.h"
 #include "vtkImageCast.h"
 
-//----------------------------------------------------------------------------
 
+//----------------------------------------------------------------------------
 
+//Global variables
+
+//Magic numbers, taken from PlusB-bin\PlusLibData\ConfigFiles\ImageProcessing\PlusDeviceSet_Server_TransverseProcessEnhancerConfig.xml
+const int numberOfScanLinesDefault = 200;
+const int numberOfSamplesPerScanLineDefault = 210;
+const double radiusStartMmDefault = 50.0;
+const double radiusStopMmDefault = 120.0;
+const int thetaStartDegDefault = -24;
+const int thetaStopDegDefault = 24;
+
+int argc;
+char **argv;
+
+//Variables that will be taken form command line arguments
+std::string inputFileName;
+std::string configFileName;
+std::string outputFileName;
+std::string linesImageFileName;
+std::string intermediateImageFileName;
+std::string processedLinesImageFileName;
+
+int numberOfScanLines = NULL;
+int numberOfSamplesPerScanLine = NULL;
+
+double radiusStartMm = NULL;

```

Comment:
double should be initialized with 0.0
(but you'll probably remove this anyway, as you'll set parameters using a config file)